src/app/screens/LogIn/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import LogInLayout from './layout';
+import './styles.css';
+
+class LogIn extends React.Component {
+  validateLogIn = () => {
+    document.getElementById('password-validation').style.display = 'none';
+    document.getElementById('user-validation').style.display = 'none';
+    const user = document.getElementById('user').value;
+    const password = document.getElementById('password').value;
```

Comment:
declare `user`, `user-validation`, `password` and `password-validation` as constants in `screens/Login/constants.js`

and use the constants here and in `LoginLayout`